README.rst

```diff
@@ -20,6 +20,53 @@
         :target: http://ntifakestatsd.readthedocs.io/en/latest/?badge=latest
         :alt: Documentation Status
 
-nti.fakestatsd is a testing client for StatsD.
+nti.fakestatsd is a testing client for verifying StatsD metrics emitted by perfmetrics.
+
+It's easy to create a new client for use in testing.
+
+.. code-block:: bash
```

Comment:
Try `pycon` instead of `bash`. But really, `code-block` shouldn't be needed at all, doctest blocks get special rendering already when set up correctly.

If we're going to have this level of usage documentation, we need to test it. I'll go ahead and push a commit to this branch to do so.